sql/pgmq--0.25.0--0.26.0.sql

```diff
@@ -0,0 +1,104 @@
+-- Dropping rust impls
+DROP FUNCTION pgmq.read(text, integer, integer);
+DROP FUNCTION pgmq.read_with_poll(text, integer, integer, integer, integer);
+
+-- Adding message record type, required for the functions
+CREATE TYPE pgmq.message_record AS (
+    msg_id BIGINT,
+    read_ct INTEGER,
+    enqueued_at TIMESTAMP WITH TIME ZONE,
+    vt TIMESTAMP WITH TIME ZONE,
+    message JSONB
+);
+
+-- Adding PL/pgSQL impls
+-- Copy-pasted from pgrx schema
+CREATE FUNCTION pgmq.read(
+    queue_name TEXT,
+    vt INTEGER,
+    qty INTEGER
+)
+RETURNS SETOF pgmq.message_record AS $$
+DECLARE
+    sql TEXT;
+BEGIN
+    sql := FORMAT(
+        $QUERY$
+        WITH cte AS
+        (
+            SELECT msg_id
+            FROM pgmq.queue_%s
+            WHERE vt <= clock_timestamp()
+            ORDER BY msg_id ASC
+            LIMIT $1
+            FOR UPDATE SKIP LOCKED
+        )
+        UPDATE pgmq.queue_%s
+        SET
+            vt = clock_timestamp() + interval '$2 seconds',
+            read_ct = read_ct + 1
+        WHERE msg_id in (select msg_id from cte)
```

Comment:
This part of the read query changed in https://github.com/tembo-io/pgmq/pull/113